web/src/routes.rs

```diff
@@ -2,13 +2,15 @@ use axum::routing::get;
 use axum::routing::post;
 
 use crate::{
+    admin,
     pages::{home, landing, user_dashboard},
     users, AppState,
 };
 
 pub fn routes(app_state: AppState) -> axum::Router {
     axum::Router::new()
         .route("/", get(home))
+        .nest("/_", admin::routes())
```

Comment:
This allows us to define all the admin routes in the admin module just to organize things a bit!

We might break down more things like this later!